packages/js/src/first-time-configuration/tailwind-components/steps/social-profiles/social-input-person-section.js

```diff
@@ -47,63 +46,17 @@ function SocialInputPersonSection( { socialProfiles, errorFields, dispatch, canE
 		<div>
 			{
 				// No person has been selected in step 2
-				personId === 0 && <Fragment>
-					<Alert type="info">
-						{
-							createInterpolateElement(
-								sprintf(
-									__(
-										// translators: %1$s and %2$s are replaced by opening and closing <b> tags, %3$s and %4$s are replaced by opening and closing anchor tags
-										"%1$sImportant%2$s: Please select a name in step 2 for this step to be effective.",
-										"wordpress-seo"
-									),
-									"<b>",
-									"</b>"
-								),
-								{
-									b: <b />,
-								}
-							)
-						}
-					</Alert>
-					<p>
-						{
-							createInterpolateElement(
-								sprintf(
-									__(
-										// translators: %1$s and %2$s are replaced by opening and closing anchor tags
-										"In this step, you need to add the personal social profiles of the person your site represents. To do that, you should go to the %1$sUsers%2$s > Profile page in a new browser tab. Alternatively, ask the user or an admin to do it if you are not allowed.",
-										"wordpress-seo"
-									),
-									"<a>",
-									"</a>"
-								),
-								{
-									b: <b />,
-									// eslint-disable-next-line jsx-a11y/anchor-has-content
-									a: <a id="yoast-configuration-workout-user-page-link-1" href={ window.wpseoFirstTimeConfigurationData.usersPageUrl } target="_blank" rel="noopener noreferrer" />,
-								}
-							)
-						}
-					</p>
-					<p>
-						{
-							addLinkToString(
-								sprintf(
-									__(
-										// translators: %1$s and %2$s are replaced by opening and closing anchor tags.
-										"On the %1$sUsers%2$s page, hover your mouse over the username you want to edit. Click ‘Edit’ to access the user’s profile. Then, scroll down to the ‘Contact info’ section (see screenshot below) and fill in the URLs of the personal social profiles you want to add.",
-										"wordpress-seo"
-									),
-									"<a>",
-									"</a>"
-								),
-								window.wpseoFirstTimeConfigurationData.usersPageUrl,
-								"yoast-configuration-workout-user-page-link-2"
+				personId === 0 && <Alert type="info" className="yst-mt-5">
+					{
+						sprintf(
+							__(
+								// translators: %1$s and %2$s are replaced by opening and closing <b> tags, %3$s and %4$s are replaced by opening and closing anchor tags
+								"Please select a name in step 2 for this step to be effective.",
```

Comment:
I'd love to be more specific here: 
"Please select a name in the site representation step for this step to be effective."

In that case, keep the translator comment (but move it to the right place) and explain that the translation for `site representation` should be consistent with the translation used for the step header.